earth_enterprise/src/common/SConscript

```diff
@@ -175,9 +175,9 @@ env.test('tile_resampler_unittest',
          'tile_resampler_unittest.cpp',
          LIBS=['gecommon', 'gtest', 'qt-mt'])
 
-env.test('verref_storage_unittest',
```

Comment:
Note: There's an RPM spec file, and commands to build googletest from source with the `devtoolset-2` tool chain on RHEL 6 in the work-in-progress pull request #412.  Installing the RPM avoids the need to disable `gtest` tests.